apps/bfd-server/bfd-server-war/target/test-classes/fakeDrugOrg.tsv

```diff
@@ -0,0 +1,2 @@
+PRODUCTID	PRODUCTNDC	PRODUCTTYPENAME	PROPRIETARYNAME	PROPRIETARYNAMESUFFIX	NONPROPRIETARYNAME	DOSAGEFORMNAME	ROUTENAME	STARTMARKETINGDATE	ENDMARKETINGDATE	MARKETINGCATEGORYNAME	APPLICATIONNUMBER	LABELERNAME	SUBSTANCENAME	ACTIVE_NUMERATOR_STRENGTH	ACTIVE_INGRED_UNIT	PHARM_CLASSES	DEASCHEDULE	NDC_EXCLUDE_FLAG	LISTING_RECORD_CERTIFIED_THROUGH
```

Comment:
Just for my clarification, were these files created by you or generated?

Reply:
I created the files

Reply:
Won't it get erased every time you run mvn since target will get regenerated?

Reply:
Ah, good find. Yes we should move this out of the `target` folder since that should only be for generated code.